client/lib/site/index.js

```diff
@@ -121,6 +121,10 @@ Site.prototype.updateComputedAttributes = function() {
 		! this.is_vip &&
 		isHttps( this.options.unmapped_url )
 	);
+	this.is_customizable = !! (
+		this.capabilities &&
+		this.capabilities.edit_theme_options
+	);
```

Comment:
I would have preferred to not copy the `isCustomizable` function into a property here, but I didn't manage to have our selector fetch a proper `Site` object (instead of the plain `Object` we get from the state tree) without also breaking the tests.